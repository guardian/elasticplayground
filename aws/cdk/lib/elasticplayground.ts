import { join } from 'path';
import type { GuStackProps } from '@guardian/cdk/lib/constructs/core';
import { GuStack } from '@guardian/cdk/lib/constructs/core';
import type { App } from 'aws-cdk-lib';
import { CfnInclude } from 'aws-cdk-lib/cloudformation-include';
import {GuEc2App} from "@guardian/cdk";
import {GuVpc} from "@guardian/cdk/lib/constructs/ec2";

export class Elasticplayground extends GuStack {
	constructor(scope: App, id: string, props: GuStackProps) {
		super(scope, id, props);

		const vpc = GuVpc.fromIdParameter(this, "primary-vpc", {

		})
	}
}
