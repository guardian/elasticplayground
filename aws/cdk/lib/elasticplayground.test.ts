import { App } from 'aws-cdk-lib';
import { Template } from 'aws-cdk-lib/assertions';
import { Elasticplayground } from './elasticplayground';

describe('The Elasticplayground stack', () => {
	it('matches the snapshot', () => {
		const app = new App();
		const stack = new Elasticplayground(app, 'Elasticplayground', {
			stack: 'playground',
			stage: 'TEST',
		});
		const template = Template.fromStack(stack);
		expect(template.toJSON()).toMatchSnapshot();
	});
});
