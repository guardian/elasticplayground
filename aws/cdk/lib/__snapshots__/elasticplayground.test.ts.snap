// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Elasticplayground stack matches the snapshot 1`] = `Object {}`;
